### PR TITLE
Adding BlueBot

### PR DESCRIPTION
**Pull request by thibautbessone**
This pull request was made by the Gatsby Branch Editor.

### CODE REVIEW

File: data/en/bots/268420199370194944.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  owner: thibautbessone
+  repo: DiscordBlueBot
+images: []
+link: >-
+  https://discordapp.com/oauth2/authorize?client_id=268420199370194944&scope=bot&permissions=-1
+nsfw: false
+pagename: BlueBot
+prefix: '! (Customizable)'
+support: 'https://discord.gg/rSekkJv'
+---
```
Comment (Andre601) [inline data/en/bots/268420199370194944.md]: Not even a long description?

Comment (thibautbessone) [reply data/en/bots/268420199370194944.md]: I assumed you could edit the description later, I can't really do it right now - if you actually can't edit the description later, reject this PR and I'll make another one when I have time

Comment (Andre601) [reply data/en/bots/268420199370194944.md]: You can, but that would require another PR for updating it.
So it's completely your choice.

Comment (thibautbessone) [reply data/en/bots/268420199370194944.md]: Alright - please accept this one then, I don't really know when I'll take the time to write a more exhaustive description. Thanks !

**Comment by lepon01** [comment]
Hot drama

You can edit the file by editing your fork.

If you are too lazy to run a `git clone`, you can edit it here https://github.com/thibautbessone/Discord_Fork/edit/master/data/en/bots/268420199370194944.md?pr=%2FTerminal%2FDiscord_Fork%2Fpull%2F174

**Pull request merged**